Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/BackLighting.azsli

```diff
@@ -56,9 +56,9 @@ float3 GetBackLighting(Surface surface, LightingData lightingData, float3 lightI
             // https://colinbarrebrisebois.com/2011/03/07/gdc-2011-approximating-translucency-for-a-fast-cheap-and-convincing-subsurface-scattering-look/
 
             {
-                thickness = max(transmissionDistance, surface.transmission.thickness);
-                float transmittance = pow( saturate( dot( lightingData.dirToCamera, -normalize( dirToLight + surface.normal * transmissionParams.z ) ) ), transmissionParams.y ) * transmissionParams.w;
-                float lamberAttenuation = exp(-thickness * transmissionParams.x) * saturate(1.0 - thickness);
+                thickness = max(transmissionDistance, transmission.thickness);
+                float transmittance = pow( saturate( dot( lightingData.dirToCamera, -normalize( dirToLight + surface.normal * transmission.GetDistortion() ) ) ), transmission.GetPower() ) * transmission.GetScale();
+                float lamberAttenuation = exp(-thickness * surface.transmission.GetAttenuationCoefficient()) * saturate(1.0 - thickness);
```

Comment:
Should this be? `transmission.GetAttenuationCoefficient()`?

Reply:
transmission is a cached reference to surface.transmission, it shouldn't behave any differently.

Reply:
My bad, fixed now!